eox_nelp/one_time_password/view_decorators.py

```diff
@@ -55,8 +56,24 @@ def wrapper(request):
             if not proposed_user_otp == cache.get(user_otp_key):
                 return HttpResponseForbidden(reason="Forbidden - wrong code")
 
+            save_successfull_phone_validation(request.user)
+            logger.info("Saved sucessfull validated otp for %s*****", user_otp_key[:-5])
```

Comment:
Do we need this log ? there is a log six line before that indicates that the validation process is in progress, so that tell us that the process started and we can check the model  to verify if that has finished